src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -19,27 +19,43 @@
 
 abstract class Extractor
 {
-    const DEFAULT_MAX_TRIES = 5;
+    public const DEFAULT_MAX_TRIES = 5;
 
-    /** @var \PDO */
+    /**
+     * @var \PDO
+     */
```

Comment:
oneline if you feel like fixing it. I wouldn't demand it, it just would be nice.